security/pfSense-pkg-acme/files/usr/local/www/acme/acme_accountkeys_edit.php

```diff
@@ -204,15 +204,14 @@ function customdrawcell_actions($object, $item, $itemvalue, $editable, $itemname
 	$pconfig['acmeserver'],
 	form_keyvalue_array($a_acmeserver)
 ))->setHelp('The ACME server which will be used to issue certificates using this key.%1$s' .
-	'Use testing servers until certificate validation works, then switch to production.%1$s' .
-	'Let\'s Encrypt ACMEv1 servers no longer allow new registrations, and in June 2021 they will be completely disabled.%1$s%1$s', '<br/>');
+	'Use testing servers until certificate validation works, then switch to production.%1$s%1$s', '<br/>');
 
 $section->addInput(new \Form_Input(
 	'email',
 	'E-Mail Address',
 	'text',
 	$pconfig['email']
-))->setHelp('The e-mail address to register for this key. This is used by Let\'s Encrypt to send automated certificate expiration notices.');
+))->setHelp('The e-mail address to register for this key. This is used to send automated certificate expiration notices.');
```

Comment:
This should at least retain a reference to it being the CA that sends these _not_ the firewall. It's easy for users to be confused otherwise.

e.g. "The ACME Server/CA may send messages to this address, such as automated notices for expiring certificates."